content/applications/hr/payroll/contracts.rst

```diff
@@ -0,0 +1,249 @@
+=========
+Contracts
+=========
+
+Every employee in Odoo is required to have a contract in order to be paid. A contract outlines the
+terms of an employee's position, their compensation, their working hours, and any other details
+about their position.
+
+.. important::
+   Contract documents (PDF's) are uploaded and organized using the *Documents* application, and are
+   signed using the *Sign* application. Ensure these applications are installed in order to send and
+   sign contracts. Please refer to the :doc:`/applications/finance/documents` and
+   :doc:`/applications/finance/sign` documentation.
+
+To view the employee contracts, go to the :menuselection:`Payroll app --> Employees --> Contracts`
+from the top menu.
+
+.. image:: contracts/payroll-contracts.png
+   :align: center
+   :alt: Navigate to the contracts dashboard in Payroll.
+
+All employee contracts, and their current contract status, are displayed in a default kanban view.
+The default view displays both running contracts and contracts that need action. Expired and
+canceled contracts are hidden in the default view.
+
+.. image:: contracts/contracts-overview.png
+   :align: center
+   :alt: Contracts dashboard view showing running contracts and contracts with issues.
+
+.. note::
+   The list of contracts in the *Payroll* application matches the list of contracts in the
+   *Employees* application. The default contracts view in the *Payroll* application displays running
+   contracts and contracts needing attention, while the default contracts view in the *Employees*
+   application displays all contracts in a kanban view, organized by their stage, regardless of
+   status. All contracts can be viewed by changing the filters.
+
+Create a new contract
+=====================
+
+In order for an employee to be paid, an active contract is required. If a new contract is needed,
+click the :guilabel:`Create` button on the contracts dashboard.
+
+.. image:: contracts/new-contract-button.png
+   :align: center
+   :alt: Click the NEW smart button to create a new contract.
+
+A contract form appears where the information can be entered. Required fields are underlined in
+bold.
+
+Required fields
+---------------
+
+.. image:: contracts/required-fields.png
```

Comment:
ideally we show a real-life use case on form views like this to better educate the reader (instead of a blank form)